src/data/faq.json

```diff
@@ -972,6 +972,16 @@
                         "textblock": [
                             "<ul><li><b> Create a rapid test profile:</b> In order to create a rapid test profile, tap on 'Manage Your Tests' on the Status tab ('Start Screen' before version 2.12) of your CWA (or on 'Register Test' for app versions until 2.5). There you will find the area 'Create Rapid Test Profile', where you can enter your personal data, such as name, address and telephone number. When everything is entered, you can save the data. The Corona-Warn-App then creates a QR code from the data that you can show at the test sites.</li><li><b>Delete rapid test profile (iOS):</b> You can delete a rapid test profile by going to your rapid test profile in the rapid test area of the Corona-Warn-App and selecting 'Edit' and then 'Delete Rapid Test Profile'. (Earlier versions offered the menu item 'Remove Rapid Test Profile' for this function.)</li><li><b>Delete rapid test profile (Android):</b> You can delete a rapid test profile by going to your rapid test profile in the rapid test area of the Corona-Warn-App and selecting 'Delete Rapid Test Profile' in the 3-dot-icon. (Version 2.5 and earlier offered the menu item 'Remove' for this function.)</li><li><b>Edit rapid test profile:</b> If you intend to edit a rapid test profile in app versions until 2.5, you can first remove (delete) the profile and then create a new one. Starting with version 2.6 of the app, a rapid test profile can be edited, so it is not necessary to delete the profile in order to make changes to it.</li></ul>"
                         ]
+                    },
+                    {
+                        "title": "How efficiently does the test I use detect an infection?",
+                        "anchor": "schnelltesttest",
+                        "active": false,
+                        "textblock": [
+                            "<a href='https://schnelltesttest.de/' target='_blank' rel='noopener noreferrer'>schnelltesttest.de</a> is a German open source project that provides you with an easy way to retrieve information about the SARS-COV-2 antigen rapid tests evaluated by the Paul Ehrlich Institute.",
+                            "After specifying a test, which can be done by scanning the barcode or entering the manufacturer's name, you will be shown the percentage of infected persons with very high viral loads and the percentage of infected persons with all viral loads.",
+                            "This is an external project, which is not related to the Corona-Warn-App. The Corona-Warn-App is not responsible for the content of the mentioned page."

```

Comment:
"The Corona-Warn-App is not responsible for the content of the mentioned page." => 

"The publisher of the Corona-Warn-App is not responsible for the content of the mentioned page."

Reply:
Changed it in https://github.com/corona-warn-app/cwa-website/pull/2533/commits/3796aeb0d7637de624daf0310c8200a95d0dae96, thanks for your suggestion!